TESTING.md

```diff
@@ -5,64 +5,52 @@
 - [Integration tests](#integration-tests)
 - [Rake Tasks](#rake-tasks)
 
-## Syntax and style
+## Requirements
 
-### Requirements:
+- [ChefDK](https://downloads.chef.io/chefdk)
+- [Vagrant](https://www.vagrantup.com/)
+- [Packer](https://www.packer.io/)
+- A supported macOS hypervisor:
+  - [Parallels](https://www.parallels.com/landingpage/pd/general/)
+  - [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+  - [VMWare Fusion](https://www.vmware.com/products/fusion.html)
 
-- `cookstyle` and `foodcritic` (both are shipped with the [ChefDK](https://downloads.chef.io/chefdk))
+## Syntax and style
 
-### Running the tests
+- `cookstyle` and `foodcritic`
 
 Syntax testing is pretty straight forward. At the root of the cookbook, run:
 
 ```shell
-$ cookstyle
-```
-
-and
-
-```
-$ foodcritic .
+cookstyle
+foodcritic .
 ```
 
 ## Unit tests
 
-### Requirements:
-
-- [RSpec](http://rspec.info/)
-- [ChefDK](https://downloads.chef.io/chefdk) (optional)
-
 For unit tests, we focus on testing the library files, which are written
 in pure Ruby and tested with RSpec. The library files contain most of the core
 business logic for each of the custom resources and are used as either mixins
 for the custom resources or contain classes that act as helpers, except with
 explicit namespacing. Some libary files are heavily unit tested, others are definitely
-missing much-needed unit test coverage.
-
-### Running the tests
+missing much-needed unit test coverage. The following command syntax assumes
+you've made the ChefDK `rspec` your default by running `chef shell-init bash`
+(https://docs.chef.io/ctl_chef.html#chef-shell-init)
 
 Clone this repo and in the root of the cookbook, run:
 
 ```shell
-$ rspec spec
+rspec spec
 ```
 
 To run the unit tests in a specific file:
 
 ```shell
-$ rspec spec/unit/libraries/xcode_spec.rb
+rspec spec/unit/libraries/xcode_spec.rb
 ```
 
 ## Integration tests
 
-###  Requirements:
-
-- [ChefDK](https://downloads.chef.io/chefdk)
-- A virtual machine provider (we use [Parallels](https://www.parallels.com/landingpage/pd/general/), but [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [VMWare Fusion](https://www.vmware.com/products/fusion.html) should be fine)
-- [Vagrant](https://www.vagrantup.com/)
-- macOS Vagrant base boxes running (we're currently testing 10.11.6, 10.12.6, and 10.13.3)
-- [Packer](https://www.packer.io/) (recommended for box building)
-
```

Comment:
Why were these references removed when they are generally the preferred tools used to build base boxes?

Reply:
These were moved to lines 8-16 as we want requirements towards the top.